packages/jetbrains-plugin/src/test/kotlin/com/mongodb/jbplugin/inspections/correctness/JavaDriverMongoDbFieldDoesNotExistTest.kt

```diff
@@ -27,6 +27,38 @@ public FindIterable<Document> exampleFind() {
             .getCollection("myCollection")
             .find(eq(<warning descr="Field \"nonExistingField\" does not seem to exist in collection.">"nonExistingField"</warning>, "123"));
 }
+// Additional tests to verify INTELLIJ-317
+public MongoCursor<Document> exampleFind1() {
```

Comment:
No strong feelings here and happy to keep it as-is if it follows established naming patterns, but would it make sense to name these methods a bit more descriptively? E.g. `findIterator_nonExistentField_showsWarning` - that way a reader doesn't have to read the entire test case source (which is admittedly not very long) to understand what is being tested, under what conditions, and what the expected outcome is.